python/its-client/its_client/configuration.py

```diff
@@ -21,10 +21,17 @@ def build(args=None) -> ConfigParser:
         help="port of the MQTT broker",
     )
     parser.add_argument(
-        "--mqtt-tls-port",
+        "--mqtt-tls",
```

Comment:
The `--mqtt-*` prefix is continuing the trend of the existing options.

Changing existing options is not nice, as users would get confused and existing
tooling around the program would break. So, we want to keep existing options
unchanged. For the `--mqtt-tls-port` one, however, we know it never worked, so
nobody ever used it, so it's fine with changing it (and I'm totally OK with
dropping the `-T` short option).

So, for new options, we want to keep the same semantics, so we keep the `--mqtt-`
prefix.

Additionally, we are going to add a few options related to logging (see #48), and
those are going tp be prefixed with `--log-`, as a kind of a cheap namespace for
options.

The alternative, which I would be totally fine with, is to keep a single option,
`--config-file`, and only use settings from the config file. Indeed, starting the
its-client manually with tons of options is probably not very practical, while
editing a `.ini` config file is much user-friendly.

Reply:
That's what versions and deprecation are for ;)
It would also be aligned with the configuration options present in the configuration file
But fine for now, I was just opening the discussion as it triggered me

The best way is indeed to use the configuration file